fastchat/train/train.py

```diff
@@ -248,6 +248,12 @@ def train():
         cache_dir=training_args.cache_dir,
     )
     model.config.use_cache = False
+    if training_args.model_max_length > 2048:
```

Comment:
do not use hardcoded 2048. read it from `model.config` with https://github.com/lm-sys/FastChat/blob/476491066a9b7f4e91abfc19f9d9801d38aa56cf/fastchat/utils.py#L287